drs-auth_client/lib/drs/auth_client/client.rb

```diff
@@ -13,12 +13,13 @@ def initialize(auth_token)
         @auth_token = auth_token
       end
 
-      def get(path)
+      def get(path, options = {})
```

Comment:
No that makes sense to me. I was trying to keep it generic but you're right - they are getting passed straight in as params. Will change.